usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -103,6 +103,10 @@ def filters(self):
     def def_codes(self):
         return self.filters["def_codes"]
 
+    @property
+    def award_type_codes(self):
```

Comment:
This property is defined in `AwardTypeMixin`. I think it will be OK to include in the CountBase definition